catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -219,7 +219,7 @@ private void validateMigrations(Jdbi jdbi, MigrationConfiguration conf) throws I
           "There are pending migrations to be run on the database."
               + " Please backup your data and run `./bootstrap/bootstrap_storage.sh migrate-all`."
               + " You can find more information on upgrading OpenMetadata at"
-              + " https://docs.open-metadata.org/deployment/upgrade");
+              + " https://docs.open-metadata.org/deployment/upgrade ");
```

Comment:
Is this intend?

Reply:
Yes, it is as link take the Inverted Commas with him while click